specification/storage/resource-manager/Microsoft.Storage/stable/2021-04-01/examples/StorageAccountSetBlobInventoryPolicy.json

```diff
@@ -0,0 +1,154 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res7687",
+    "accountName": "sto9699",
+    "blobInventoryPolicyName": "default",
+    "api-version": "2021-04-01",
+    "monitor": "true",
+    "properties": {
+      "properties": {
+        "policy": {
+          "enabled": true,
```

Comment:
Just to confirm, the blob inventory policy don't have "Version" any more.
In the original GA design, it should have version as "1.0".